### ISSUE TITLE
[Feature request] Channel alert: make alert text optional

### ISSUE THREAD

**Issue by Mikaela** [ProgVal/Limnoria#1478]
In my opinion `alert` is a bit weird command name and I mostly see `ops` used for this purpose, it's also weird that the command demands text. Thus I have Aka "ops" for it (`aka add ops channel alert !ops $*`, but I am pointed out that this doesn't work in private.

I think the command shouldn't require text and inform ops that their attention is desired when arguments aren't given on channel, this would also allow making `aka add ops channel alert $*` which would work in private also.

> The current (running) version of this Limnoria is 2021.06.15, running on Python 3.9.2 (default, Feb 28 2021, 17:03:44)  [GCC 10.2.1 20210110].  The newest versions available online are 2021.06.15 (in master), 2021.06.25 (in testing).

**Comment by ProgVal** [comment]
> Thus I have Aka "ops" for it (`aka add ops channel alert !ops $*`, but I am pointed out that this doesn't work in private.

Yes, because it's missing the first argument. either remove `!opts` or add a first argument before

**Comment by Mikaela** [comment]
> `(alert [<channel>] <text>) -- Sends <text> to all the users in <channel> who have the <channel>,op capability.`

My preferred solution is making `<text>` optional so I can have an aka without arguments, so when it was called without arguments it would just tell `Alert to all #test ops (from <nick>)`

If I had an aka without the `!ops`, running it without arguments would give the help text (or just syntax depending on configuration), which I consider as breaking user expectations.
